packages/vue/src/modal/modal-slot-props.ts

```diff
@@ -0,0 +1,4 @@
+export type ModalSlotProps = {
```

Comment:
```suggestion
export type VueModalProps = {
```

Just to get more clarify inside the props name. `Vue` as a prefix is not consistant with the React version but `Slot` was confusing me the first time I read the type name.

Reply:
Slot properties are a common name in Vue, that's why I choose that. [Slots](https://vuejs.org/guide/components/slots#scoped-slots).

Reply:
Then slot is totally fine :)